Flaky Test `TestLoggerPause`

### Feature Description

https://github.com/go-gitea/gitea/actions/runs/5055503857/jobs/9071789890?pr=24864

```
--- FAIL: TestLoggerPause (0.20s)
    logger_test.go:100: 
        	Error Trace:	/home/runner/work/gitea/gitea/modules/log/logger_test.go:100
        	Error:      	Not equal: 
        	            	expected: []string{}
        	            	actual  : []string{"info-level\n"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,2 +1,3 @@
        	            	-([]string) {
        	            	+([]string) (len=1) {
        	            	+ (string) (len=11) "info-level\n"
        	            	 }
        	Test:       	TestLoggerPause
```

Maybe regression from https://github.com/go-gitea/gitea/pull/24726.

### Screenshots

_No response_

--- a/modules/log/logger_test.go
+++ b/modules/log/logger_test.go
@@ -94,6 +94,7 @@ func TestLoggerPause(t *testing.T) {
 	logger.AddWriters(w1)
 
 	GetManager().PauseAll()
+	time.Sleep(50 * time.Millisecond)
 
 	logger.Info("info-level")
 	time.Sleep(100 * time.Millisecond)

